### ISSUE TITLE
[Bug] Binding converter properties in a resource dictionary requires archaic syntax

### ISSUE THREAD

**Issue by Tommigun1980** [xamarin/Xamarin.Forms#11785]
This is a follow-up to https://github.com/xamarin/Xamarin.Forms/issues/11759 that got closed before I had a chance to reply.

So TLDR:

If you bind converter properties in a resource dictionary, such as to a VM's 'ActivityTypeIdFilter' property in the following example;

```xaml
<converters:IsNotEqualConverter x:Key="XXX" Value="{Binding BindingContext.ActivityTypeIdFilter, Source={x:Reference tabRoot}}" />
```

they require you to use a reference and add a 'BindingContext dot' to the binding or it won't work. When binding against the same properties inside the content object you can not use 'BindingContext dot' or it won't work. Both incorrect permutations compile but don't work. It is very hard to remember adding 'BindingContext dot' when binding converter parameters. There may be an internal reason for it, but it is so cumbersome to use. Could the resource dictionary's context be set to the same as the content object's, or something else be done about this oddity in the bindings?

So to access the VM's 'ActivityTypeIdFilter' in a converter binding you have to write

```xaml
<converters:IsNotEqualConverter x:Key="XXX" Value="{Binding BindingContext.ActivityTypeIdFilter, Source={x:Reference viewOrPageRoot}}" />
```

while accessing the same VM's 'ActivityTypeIdFilter' inside the content object you have to write

```xaml
<Label Text="{Binding ActivityTypeIdFilter}" />
```

As I said there may be a technical reason for this, but please consider making converter bindings easier to write. This is really archaic knowledge that you just happen to find somewhere or experiment into, as it doesn't follow any logical rules with bindings.

**Comment by rmarinho** [comment]
Hi @Tommigun1980  can you please provide us with a small sample to better understand the problem? 

Thanks

**Comment by Tommigun1980** [comment]
> Hi @Tommigun1980 can you please provide us with a small sample to better understand the problem?
> 
> Thanks

Hi @rmarinho - I will do that, please leave this open for a short while until I can do the sample.

**Comment by samhouts** [comment]
@Tommigun1980 Will do! Thanks!

**Comment by samhouts** [comment]
@Tommigun1980 Do you need more time for the sample?

**Comment by Tommigun1980** [comment]
@samhouts Unfortunately I do :(

**Comment by samhouts** [comment]
No worries, we'll keep looking for it. Thanks!

**Comment by Redth** [comment]
Since we haven't heard from you in more than 30 days, we hope this issue is no longer affecting you. If it is, please reopen this issue and provide the requested information so that we can look into it further. Thank you!
